sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -46,6 +46,18 @@ def my_pipeline(a: int = 1, b: str = "default value"):
   ```
   """
 
+  def compile(self, pipeline_func, package_path, type_check=True, pipeline_conf: dsl.PipelineConf = None, generate_pipelinerun=False):
```

Comment:
can you move the `compile` function down to just above `_write_workflow` so that when we compare code changes with KFP all the methods line up

and break the line before the `generate_pipelinerun` argument to keep the line length below 120 for code style compliance and better readability

Reply:
Thanks @ckadner, I moved compile function.